### ISSUE TITLE
[Bug] Missing container image for dbt-bigquery-1.7.5

### ISSUE THREAD

**Issue by wouter** [dbt-labs/dbt-bigquery#1113]
### Is this a new bug in dbt-bigquery?

- [X] I believe this is a new bug in dbt-bigquery
- [X] I have searched the existing issues, and I could not find an existing issue for this bug

### Current Behavior

The last release version of dbt-bigquery is 1.7.5, but the image for this version is not (yet) available in the container registry.

### Expected Behavior

If version `1.7.5` is released, I expect there to be a container image at `ghcr.io/dbt-labs/dbt-bigquery:1.7.5`

### Steps To Reproduce

Create a Dockerfile with the following line:

`FROM ghcr.io/dbt-labs/dbt-bigquery:1.7.5`

Run it and see that it fails:

```
--------------------
   1 | >>> FROM ghcr.io/dbt-labs/dbt-bigquery:1.7.5
--------------------
ERROR: failed to solve: ghcr.io/dbt-labs/dbt-bigquery:1.7.5: ghcr.io/dbt-labs/dbt-bigquery:1.7.5: not found
```

### Environment

```markdown
- OS:
- Python:
- dbt-core:
- dbt-bigquery:
```

**Comment by dbeatty10** [comment]
Thanks for reporting this @wouter -- I've added this to https://github.com/dbt-labs/dbt-core/issues/7571

**Comment by dbeatty10** [comment]
Thank you again for reporting this @wouter 🏅

Since your initial report, we've released `1.7.6` and `1.7.7` and published container images for each, with the most recent tagged appropriately:

<img width="219" alt="image" src="https://github.com/dbt-labs/dbt-bigquery/assets/44704949/314f28fd-0889-4c1b-871b-246af539ba51">

We've accidentally missed versions like `1.7.5` a few times, and we're sorry about the impact it had. We recently merged https://github.com/dbt-labs/dbt-release/issues/109 to try to prevent this in the future.

I'm going to close this as "not planned" since we'd have to do a manual release to backfill this, which is an effort we'd like to avoid so we can put it towards other areas.

Because we have more recent images available now, so hopefully this isn't too much of a problem. Please let us know if it is and we can revisit it.

**Comment by khaledh** [comment]
> We recently merged https://github.com/dbt-labs/dbt-release/issues/109 to try to prevent this in the future.

@dbeatty10 What about `1.8.0b1` and `1.8.0b1`? I can find the `dbt-core` images for those, but not for `dbt-bigquery`.

**Comment by dbeatty10** [comment]
@khaledh Looking at the history [here](https://github.com/dbt-labs/dbt-bigquery/pkgs/container/dbt-bigquery/versions?filters%5Bversion_type%5D=tagged), we haven't tended to publish GCR container images for betas, and it's mostly been release candidates (RC) instead.

Can you share more about your use-case?

Looping in my colleague @dataders who helps consider our adapter-related interfaces.

**Comment by khaledh** [comment]
@dbeatty10 We are interested in the new built-in dbt [unit tests](https://docs.getdbt.com/docs/build/unit-tests) support, which is available only starting in 1.8. Our tests run in CI in a docker container, so we rely on the `dbt-bigquery` image for that.

As a workaround for now, we are using the `dbt-core:1.8.0b2` image and we added a Dockerfile step to `pip install dbt-bigquery==1.8.0b2`.
